desktop/core/src/desktop/conf.py

```diff
@@ -2174,6 +2174,28 @@ def has_raz_url():
   return bool(_get_raz_url())
 
 
+SDXAAS = ConfigSection(
+  key='sdxaas',
+  help=_("""Configuration for SDXaaS JWT support."""),
+  members=dict(
+    TOKEN_URL=Config(
+      key='token_url',
+      help=_('Comma separated host URLs to fetch token from.'),
+      type=str,
+      default='',
+    )
+  )
+)
+
+def is_sdxaas_jwt_enabled():
+  """Check if SDXaaS token url is configured"""
+  return SDXAAS.TOKEN_URL.get() != ''
```

Comment:
Nit: Take a look this one: https://stackoverflow.com/questions/9573244/how-to-check-if-the-string-is-empty

Reply:
Changed it and using `bool()` now.